batch/Makefile

```diff
@@ -84,7 +89,12 @@ deploy: push
 	kubectl delete persistentvolumeclaim --all --namespace batch-pods
 	kubectl -n default apply -f deployment.yaml
 
-test-deploy: push-test
+ifndef IN_HAIL_CI
```

Comment:
We seem to prefer ifeq ($(IN_HAIL_CI), 1) https://github.com/hail-is/hail/search?q=ifeq&unscoped_q=ifeq